### ISSUE TITLE
Install.sh using Ubuntu 14.04 is resulting in 'unexpected operator'

### ISSUE THREAD

**Issue by clintonskitson** [emccode/rexray#76]
```
root@ec2-54-219-167-85:/home/ubuntu# curl -sSL https://raw.githubusercontent.com/emccode/rexray/master/install.sh | sh -
sh: 14: [: x86_64: unexpected operator
```

I can resolve the problem using `#!/bin/bash` instead of `sh`.  Doing more testing to figure out what's going on..

**Comment by akutz** [comment]
Hi @clintonskitson, 

What version of Debian?

**Comment by akutz** [comment]
Nevermind.

**Comment by clintonskitson** [comment]
Changed it to `[ "$ARCH" = "x86_64"` and it worked for `sh`.

**Comment by akutz** [comment]
Thanks. I used to love Ubuntu. Now I hate it. This is why it failed:

```
[0]akutz@ubuntu:rexray$ ls /bin/sh
lrwxrwxrwx 1 root 4 Aug 12 17:01 /bin/sh -> dash*
[0]akutz@ubuntu:rexray$ 
```

I'm not sure when I tested it that it worked, but I have logs and screenshots showing it did. I guess I changed something at some point. I know `install.sh` has `/bin/bash` at the top, so it must be an issue with the new method of passing arguments to `install.sh` or how the static file was interpreted...

**Comment by akutz** [comment]
Closed with d8108a0cd7257c

**Comment by akutz** [comment]
```
[0]akutz@ubuntu:rexray$ ./install
Selecting previously unselected package rexray.
(Reading database ... 65583 files and directories currently installed.)
Preparing to unpack rexray-latest-x86_64.deb ...
Unpacking rexray (0.2.0+rc4-1) ...
Setting up rexray (0.2.0+rc4-1) ...

REX-Ray has been installed to /usr/bin/rexray

SemVer: 0.2.0-rc4
Binary: Linux-x86_64
Branch: v0.2.0-rc4
Commit: cc3eabe9e4d908def02168e0522db3c5e978d55d
Formed: Mon, 21 Sep 2015 10:17:02 CDT
```
